modules/features/player/src/main/res/layout/fragment_sleep.xml

```diff
@@ -61,6 +61,23 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
+        <ImageView
+            android:id="@+id/sleepTimeSettings"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="24dp"
+            android:clickable="true"
+            android:contentDescription="@string/player_sleep_timer_settings"
+            android:focusable="true"
+            android:minWidth="24dp"
+            android:minHeight="24dp"
+            android:padding="8dp"
+            android:src="@drawable/sleep_timer_cog"
+            app:layout_constraintBottom_toBottomOf="@+id/sleepTitle"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/sleepTitle"
+            app:tint="?attr/player_contrast_01" />
```

Comment:
> I noticed the existing sleep timer bottom sheet dialog does not match with the new specs in figma. I will address these changes in another PR

The current design matches the Playback Effects dialog so I think don't think we need worry about matching the header at the moment. 

- Is it me, or does the settings icon seem a bit too far out to the left?
- Could we please add a ripple to the press.

Reply:
For this, I aligned the settings button with the plus button to be in the same vertical line and added the ripple effect. Could you check this again?